# REQUIRED - Name of the map
string name

# REQUIRED - ROS topic where the map is being published
string topic

# Name of EMR parent node
string parent = ""